src/platform/user/profile/vet360/containers/CopyMailingAddress.jsx

```diff
@@ -68,7 +69,7 @@ export function mapStateToProps(state, ownProps) {
     }
     if (useNewAddressForm) {
       return isEqual(
-        pick(mailingAddress, ADDRESS_PROPS),
+        pickBy(pick(mailingAddress, ADDRESS_PROPS), e => !!e),
```

Comment:
this removes undefined/null values from the mailing address before comparing it to the residential address.

Reply:
Can you add a unit test for this?

Reply:
Done. Thanks for keeping me honest :)